Complete documentation of "Variables" (#18)
atolye15/scss-starter: docs/core/variables.md

---
id: variables
title: Variables
---

You can find and customize these variables in our `abstracts/_variables.scss` file.

## Transition

### `$g-transition-duration`

Global transition duration.

```scss
$g-transition-duration: 0.4s !default;
```

## Links

> Style anchor elements. More variables for anchors can be found in the [Colors](core/colors.md) section.

Reviewer: ```suggestion
Anchor element styles. More variables for anchors can be found in the [Colors](core/colors.md) section.
```

### `$g-link-transition`

The transition value for the global anchor elements.

```scss
$g-link-transition: null !default;
```

### `$g-link-hover-decoration`

> The `text-decoration` value for the global anchor elements in `:hover` state.

Reviewer: ```suggestion
The `text-decoration` value for the global anchor elements in the `:hover` state.
```

```scss
$g-link-hover-decoration: underline !default;
```

## Typography

### `$g-font-size-root`

> Font size of the `:root` element. It effects the value of `rem`, which is used for as well font sizes, paddings and margins.

Reviewer: ```suggestion
Font size of the `:root` element. `$g-font-size-root` effects the value of `rem`, which is used for as well font sizes, paddings, and margins.
```
